pkg/quota/evaluator/core/pods.go

```diff
@@ -118,23 +116,6 @@ func (p *podEvaluator) Constraints(required []api.ResourceName, item runtime.Obj
 		return fmt.Errorf("Unexpected input object %v", item)
 	}
 
-	// Pod level resources are often set during admission control
```

Comment:
During validating admission control, nothing is changed.
So we don't need to check it the second time.